admin_interface/templatetags/admin_interface_tags.py

```diff
@@ -128,3 +128,12 @@ def admin_interface_filter_removal_link(changelist, list_filter):
             "title": title,
         }
     )
+
+
+@simple_tag()
+def has_multiple_tabs(adminform, inline_forms):
```

Comment:
Could you please rename this to `admin_interface_use_changeform_tabs`?

Reply:
updated